### PR TITLE
feat: auth mgmt in state + redirect to merch onboarding/dashboard

### PR DESCRIPTION
**Pull request by onish-garg**

### CODE REVIEW

File: frontend/src/routes.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const base = 'https://speedy-anthem-217710.an.r.appspot.com';
```
Comment (harshil93) [inline frontend/src/routes.js:1]: why is this needed ?

Comment (onish-garg) [reply frontend/src/routes.js:1]: For production, it is not needed. During development, the ReactJS app and Spring app can be on different domains - having the base URL for APIs at one place comes in handy then.
